.circleci/config.yml

```diff
@@ -2,18 +2,6 @@
 version: 2.1
 
 executors:
-  py2-mongo:
-    docker:
-      # This image uses the oldest version of many dependencies
-      - image: girder/girder_test:latest-py2
-      # Use the oldest supported MongoDB
-      - image: circleci/mongo:3.6-ram
```

Comment:
We happened to be leveraging the py2 environment to also test our minimum supported mongodb version. We should come up with an alternative approach. ping @brianhelba

Reply:
Fixed this by always running against the oldest MongoDB. It's far more likely to see breakages due application code using unsupported new features than it is to have the MongoDB API break on a new release (so long as we continue to track supported releases as the oldest acceptable MongoDB version).